api/src/main/resources/db/migration/V2__CreateMemeTable.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE memes (
+    ID SERIAL NOT NULL PRIMARY KEY,
+    memeName TEXT UNIQUE,
+    topText TEXT,
+    bottomText TEXT
```

Comment:
Related to your comment, yes, these should be `VARCHAR` types instead.  The size should be limited for sanity, especially the text that goes onto the image, since too much will make it useless.